.github/workflows/ttnn-stress-tests-impl.yaml

```
@@ -26,7 +26,11 @@ on:
 jobs:
   ttnn:
     name: ttnn stress tests ${{ inputs.arch }} ${{ inputs.runner-label }}
-    runs-on: ["in-service", "cloud-virtual-machine", "${{ inputs.runner-label }}"]
+    runs-on:
+      - "in-service"
+      - "${{ inputs.runner-label }}"
+      - "${{ (inputs.runner == 'config-tg' || inputs.runner == 'config-6u') && 'bare-metal' || 'cloud-virtual-machine' }}"
```

```suggestion
      - "${{ (inputs.runner-label == 'config-tg' || inputs.runner-label == 'topology-6u') && 'bare-metal' || 'cloud-virtual-machine' }}"
```

I also don't see `config-tg` or `topology-6u` listed as available runner-labels in .github/workflows/ttnn-stress-tests.yaml